react/features/base/participants/middleware.js

```diff
@@ -37,30 +39,15 @@ MiddlewareRegistry.register(store => next => action => {
         break;
 
     case KICK_PARTICIPANT:
-        if (typeof APP !== 'undefined') {
-            APP.UI.emitEvent(UIEvents.USER_KICKED, action.id);
```

Comment:
I think you can remove the USER_KICKED event and listener.

Reply:
Indeed, I've noticed that yesterday and will remove those for kick and mute.